include/seqan3/core/type_traits/iterator.hpp

```diff
@@ -127,34 +127,6 @@ struct size_type<it_t>
  * \tparam it_t The type to operate on.
  * \see seqan3::iterator_tag_t
  *
- * <table>
- *   <tr>
- *     <th>Modelled concept</th>
```

Comment:
why did you remove this?

Reply:
The links were all broken, because there is no documentation for the concepts any more. It will appear once there is on cppreference, but making "conditional" links doesn't work inside the html table (why no markdown!?). 
I tried changing it to markdown, but then it looked different. Then I realised the whole data structure is very broken and that we should remove it. I made a card for it https://github.com/seqan/seqan3/projects/3#card-27178357

Reply:
(its also detail)